game_patch/multi/network.cpp

```diff
@@ -1043,7 +1038,13 @@ CodeInjection multi_io_process_packets_injection{
             int len = regs.edi;
             auto& addr = *addr_as_ref<rf::NetAddr*>(stack_frame + 0xC);
             auto player = addr_as_ref<rf::Player*>(stack_frame + 0x10);
-            process_custom_packet(data + offset, len, addr, player);
+            #ifdef HAS_PF
+            if (!pf_process_packet(data, len, addr, player)) {
+                custom_packets::process_packet(data + offset, len, addr, player);
+            }
+            #else
+            custom_packets::process_packet(data + offset, len, addr, player);
```

Comment:
it would be better if this could be not repeated - maybe use something like `bool processed = false; if PF: processed = ...`